src/runner/provider/qti.js

```diff
@@ -143,10 +143,9 @@ var qtiItemRuntimeProvider = {
                     })
                     .catch(function(err) {
                         done(); // in case of postRendering issue, we are also done
-                        self.trigger(
-                            'warning',
-                            'Error in post rendering : ' + err instanceof Error ? err.message : err
-                        );
+                        var error = new Error('Error in post rendering : ' + err instanceof Error ? err.message : err);
```

Comment:
could you please rename `err` to either `originalError` or `renderingError` so we don't mess up the 2 variable names